config/initializers/geocoder.rb

```diff
@@ -0,0 +1,4 @@
+Geocoder.configure(
+  :lookup => :bing,
+  :api_key => ENV['GEOCODER_API_KEY'],
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.